docs/guides/executor.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Executor primitive (beta)
+description: Executor is a lower-level primitive that provides more flexibility when customizing your error mitigation workflow.
+
+---
+
+# Executor primitive (beta)
+
+<Admonition type="note" title="Beta release">
+The Executor primitive is currently in beta and might not be stable. You are invited to test it and provide feedback. (((how?)))
+</Admonition>
+
+The Executor primitive is designed for quantum information scientists to run utility-scale experiments. Executor provides finer control over advanced techniques by shifting the generation of circuit variants to the server side, saving network costs and making the process explicit and customizable. With Executor, users can fine-tune error mitigation and other techniques to find the right balance of speed and quality in their utility-scale workloads.
+
+The Executor primitive benefits users who want to improve the reliability of computations on noisy quantum hardware, and are interested in capabilities such as noise learning, twirling, and expectation value calculations. With Executor, users will gain a clearer and more composable model for execution workflows, making it easier to experiment with new techniques, reproduce results, and share methods.
```

Comment:
What does "the reliability of computations on noisy quantum hardware" mean? 

Regardless, I think the previous paragraph covers the benefit of using Executor. Perhaps this one can be simplified to 

> In this initial release, Executor support focuses mostly on Pauli-twirling, noise model learning and injection, and basis changes. More support will continue to be rolled out in the near future.